_dev/docker/mono/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-slim
+FROM node:18-slim
```

Comment:
> I am a bit weary, because during my last round of CI work, I tried to upgrade a minor node version and I ended up with some test failures which was surprising.

huh that is surprising.  one would think "LTS" is relatively stable.  yeah I'll change it.